sympy/simplify/combsimp.py

```diff
@@ -217,6 +219,232 @@ def explicate(p):
                 denom_gammas.extend(l)
             elif isg is False:
                 denom_others.extend(l)
+        # =========== level >= 2 work: factor absorbtion =========
+        if level >= 2:
```

Comment:
@jksuom, removal of "level>=2 work : factor absorption" from combsimp is giving error. Also any similar function would contain gamma due to the fact that there maybe some nested gamma function which would be evaluated.
If there is any other way please suggest me. Thanks.